portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+$(document).ready(function(){
+    $('#workout-ninja-button').click(function(){
+        $('#project-pic').attr('src','images/Workout-Ninja.jpg');
+    });
+    $('#2048-button').click(function(){
+        $('#project-pic').attr('src','images/2048-image.png');
+    });
+    
+});
```

Comment:
Related to the other comment I left about the Workout-Ninja.jpg, I think it might make sense to leave this out of this PR and add it in #2 instead.

Reply:
I have those images in PR #2, so i might just remove this part and the column showing the image in this branch.